graze/pistachio
# fix for missing images when not using list view

**Summary**

The recently added list view caused a breaking change to non list view product tiles. This PR fixes that

**Notable Changes**

Addition of a distinct new class specifically for when using background images `.grid__panel-header-img-bg`

**QA**

Visit `/layouts/product-tiles` and ensure that when below 480px viewport size, the larger first product in the list has a visible product image

**Screenshots**

![screen shot 2016-11-28 at 10 35 33](https://cloud.githubusercontent.com/assets/713293/20665177/69bbf91a-b556-11e6-8947-c0ac658cb600.png)
